schema/handler.go

```diff
@@ -70,7 +70,7 @@ func (h *Handler) RegisterAdminRoutes(admin *x.RouterAdmin) {
 //
 //nolint:deadcode,unused
 //lint:ignore U1000 Used to generate Swagger and OpenAPI definitions
-type identitySchema json.RawMessage
+type identitySchema = json.RawMessage
```

Comment:
What the...

Reply:
When you define a new type (without `=`), it removes all of the underlying types methods. Using `=` means that we only have another name for the same type.

Reply:
See also https://go.dev/play/p/rZhjaze9ZYv

Reply:
https://go.googlesource.com/proposal/+/master/design/18130-type-alias.md

Reply:
The problem was that `func (m RawMessage) MarshalJSON() ([]byte, error)` and `func (m *RawMessage) UnmarshalJSON(data []byte) error` is dropped when using type definitions which in turn leads to the json marshaller treating `identitySchema` like a byte array and serializing it as a base64 string